asyncpgsa/connection.py

```diff
@@ -68,15 +43,52 @@ def execute_defaults(query):
     return query
 
 
+def placeholder_swap(input_sql, keyword_values=None):
```

Comment:
I dont think this would need to be used outside this module. What are your thoughts on naming this with an underscore? `_placeholder_swap`?